snippets/sdk/python/deployments/with_logging.py

```diff
@@ -0,0 +1,127 @@
+
+    # Create storage account
+    # https://github.com/Azure-Samples/azure-samples-python-management/tree/main/samples/storage
+    storage_client.storage_accounts.begin_create(
+        GROUP_NAME,
+        STORAGE_ACCOUNT_NAME,
+        {
+            "sku": {"name": "Standard_GRS"},
+            "kind": "StorageV2",
+            "location": "eastus",
+            "encryption": {
+                "services": {
+                    "file": {"key_type": "Account", "enabled": True},
+                    "blob": {"key_type": "Account", "enabled": True},
+                },
+                "key_source": "Microsoft.Storage",
+            },
+        },
+    ).result()
+    print("Created a storage account.\n")
+
+    # Create blob container
+    # https://github.com/Azure-Samples/azure-samples-python-management/tree/main/samples/storage
+    storage_client.blob_containers.create(
+        GROUP_NAME, STORAGE_ACCOUNT_NAME, BLOB_CONTAINER_NAME, {}
+    )
+    print("Created a blob container:\n{}")
+
+    # Create deployment with a public IP
+    deployment = nginx_client.deployments.begin_create_or_update(
+        GROUP_NAME,
+        DEPLOYMENT_NAME,
+        {
+            "sku": {"name": "preview_Monthly_gmz7xq9ge3py"},
+            "tags": {"myKey": "myValue"},
+            "location": "eastus",
+            "identity": {
+                "type": "UserAssigned",
+                "userAssignedIdentities": {identity.id: {}},
+            },
+            "properties": {
+                "networkProfile": {
+                    "networkInterfaceConfiguration": {"subnetId": subnet.id},
+                    "frontendIPConfiguration": {
+                        "publicIPAddresses": [{"id": public_ip.id}]
+                    },
+                },
+                "logging": {
+                    "storageAccount": {
+                        "accountName": STORAGE_ACCOUNT_NAME,
+                        "containerName": BLOB_CONTAINER_NAME,
+                    }
+                },
+            },
+        },
+    ).result()
+    print("Created a deployment:\n{}".format(deployment))
+
+    # Give identity blob contributor role assignment
+    # https://github.com/Azure-Samples/azure-samples-python-management/tree/main/samples/authorization
+    role_id = "ba92f5b4-2d11-453d-a403-e96b0029c9fe"
+    role_definition_id = "/".join(
+        [
+            "subscriptions",
+            SUBSCRIPTION_ID,
+            "providers/Microsoft.Authorization/roleDefinitions",
+            role_id,
+        ]
+    )
+    role_assignment = "88888888-7000-0000-0000-000000000003"
+    auth_client.role_assignments.create(
+        deployment.id,
+        role_assignment,
+        {
+            "role_definition_id": role_definition_id,
+            "principal_id": identity.principal_id,
+        },
+    )
+    print("Created a role assignment.\n")
+
+    # Delete a deployment
+    nginx_client.deployments.begin_delete(GROUP_NAME, DEPLOYMENT_NAME).result()
```

Comment:
okay so if I run this, the user may not get enough time to look at what's happening?

Also it now does not align with `create_or_update.py` where you have it commented out? was that intentional?